### ISSUE TITLE
I got confused about the transform code of computing FID

### ISSUE THREAD

**Issue by Samuel-Recovery-123** [YanzuoLu/CFLD#43]
https://github.com/YanzuoLu/CFLD/blob/9892b2fd88ec05327b6a825182388f7185daf454/models/metrics.py#L89
```
    def forward_inception(self, x):
        x = F.interpolate(x, size=(299, 299), mode='bilinear')
        x[:, 0] = x[:, 0] * (0.229 / 0.5) + (0.485 - 0.5) / 0.5
        x[:, 1] = x[:, 1] * (0.224 / 0.5) + (0.456 - 0.5) / 0.5
        x[:, 2] = x[:, 2] * (0.225 / 0.5) + (0.406 - 0.5) / 0.5
        out = self.inception_blocks(x)
        return out
```
According to the context, x is normalized to [0, 1]. How does this transform means? It seems like denormalization from standard normalization on ImageNet with std [0.229 , 0.224,  0.225] and mean [0.485, 0.456, 0.406] and normalization to [-1, 1] subsequently. But it's maybe wrong since the range of x is [0, 1]. I referred to [the official implementation code](https://github.com/mseitzer/pytorch-fid/blob/b9c18118d082cbd263c1b8963fc4221dc1cbb659/src/pytorch_fid/inception.py#L151) of FID computing and found that the image is normalized to [-1, 1] preliminarily.  Could you please explain it?

**Comment by YanzuoLu** [comment]
Hi @Samuel-Recovery-123 
After carefully investigation, I think you are right.
It's my fault not to check the code in PIDM here (https://github.com/ankanbhunia/PIDM/blob/e4f1d880772072ee464a83240d0374bf466b6772/utils/inception.py#L124).
It says the input should be [0,1] and transform like this so I just copy it to reproduce their results.
So this implementation is wrong. We should just perform normalize with mean [0.485, 0.456, 0.406] and std [0.229, 0.224, 0.225] as official torchvision doc stated (https://pytorch.org/vision/main/models/generated/torchvision.models.inception_v3.html#torchvision.models.inception_v3).
Thanks for your carefully reviewing. 
But take it easy, since FID is really not a good metric for evaluating diffusion generated content, we should pay more attention to that what we see and what we get from user studies.
